test/test_qat.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+from torch.nn import Conv2d, BatchNorm2d, ReLU
+from torch.nn._intrinsic.qat import ConvBn2d, ConvBnReLU2d
+from torch.quantization.QConfig import default_qat_qconfig
+from torch.nn import Parameter
+from common_utils import TestCase, run_tests
+from hypothesis import given
+from hypothesis import strategies as st
+from functools import reduce
+
+
+class IntrinsicQATModuleTest(TestCase):
+
+    @given(batch_size=st.integers(1, 3),
```

Comment:
FYI: There is a `tensor_conv2d` strategy in the `hypothesis utils

Reply:
I looked at `tensor_conv2d`, why don't you include generation of `pad`, `stride` in it as well? I can migrate when everything is included.